### ISSUE TITLE
Unable to start  the docker container

### ISSUE THREAD

**Issue by groenator** [Unmanic/unmanic#164]
I created a docker-compose file to start the container, setup my folder for the library, config and tmp. When I start the container I get this error:

```
 To support LSIO projects visit:
unmanic    | https://www.linuxserver.io/donate/
unmanic    | -------------------------------------
unmanic    | GID/UID
unmanic    | -------------------------------------
unmanic    |
unmanic    | User uid:    1002
unmanic    | User gid:    1002
unmanic    | -------------------------------------
unmanic    |
unmanic    | [cont-init.d] 10-adduser: exited 0.
unmanic    | [cont-init.d] 20-config: executing...
unmanic    | **** (permissions_config) Setting run user uid=1002(abc) gid=1002(abc)  ****
unmanic    | usermod: no changes
unmanic    | **** (permissions_config) Setting permissions ****
unmanic    | **** (permissions_config) Adding run user to video group ****
unmanic    | [cont-init.d] 20-config: exited 0.
unmanic    | [cont-init.d] 30-patch-nvidia: executing...
unmanic    | nvidia-smi utility not found. Probably driver is not installed.
unmanic    | [cont-init.d] 30-patch-nvidia: exited 1.
unmanic    | [cont-init.d] 99-custom-scripts: executing...
unmanic    | [custom-init] no custom files found exiting...
unmanic    | [cont-init.d] 99-custom-scripts: exited 0.
unmanic    | [cont-init.d] done.
unmanic    | [services.d] starting services
unmanic    | [services.d] done.
unmanic    | Running Unmanic from installed module
unmanic    | Traceback (most recent call last):
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 3035, in connect
unmanic    |     self._state.set_connection(self._connect())
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 3371, in _connect
unmanic    |     isolation_level=None, **self.connect_params)
unmanic    | sqlite3.OperationalError: unable to open database file
unmanic    |
unmanic    | During handling of the above exception, another exception occurred:
unmanic    |
unmanic    | Traceback (most recent call last):
unmanic    |   File "/usr/local/bin/unmanic", line 11, in <module>
unmanic    |     sys.exit(main())
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/unmanic/service.py", line 382, in main
unmanic    |     service.run()
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/unmanic/service.py", line 338, in run
unmanic    |     self.db_connection = init_db()
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/unmanic/service.py", line 193, in init_db
unmanic    |     db_connection = unmodels.Database.select_database(database_settings)
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/unmanic/libs/unmodels/basemodel.py", line 125, in select_database
unmanic    |     db.connect()
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 3038, in connect
unmanic    |     self._initialize_connection(self._state.conn)
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 2873, in __exit__
unmanic    |     reraise(new_type, new_type(exc_value, *exc_args), traceback)
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 183, in reraise
unmanic    |     raise value.with_traceback(tb)
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 3035, in connect
unmanic    |     self._state.set_connection(self._connect())
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 3371, in _connect
unmanic    |     isolation_level=None, **self.connect_params)
unmanic    | peewee.OperationalError: unable to open database file
unmanic    | Running Unmanic from installed module
unmanic    | Traceback (most recent call last):
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 3035, in connect
unmanic    |     self._state.set_connection(self._connect())
unmanic    |   File "/usr/local/lib/python3.6/dist-packages/peewee.py", line 3371, in _connect
unmanic    |     isolation_level=None, **self.connect_params)
unmanic    | sqlite3.OperationalError: unable to open database file
unmanic    |
unmanic    | During handling of the above exception, another exception occurred:
```

This is my docker-compose file:

```
version: '2.4'
services:
  unmanic:
    container_name: unmanic
    image: josh5/unmanic:latest
    ports:
        - "8880:8888"
    environment:
        - PUID=1002
        - PGID=1002
        - TZ=Europe/London
    volumes:
      - /tmp:/tmp/unmanic
      - /data/media/unmanic:/config
      - /nfs/media/Recordings:/library
    devices:
      - /dev/dri:/dev/dri
    restart: always
```

The only thing I changed is the host port because is clashing with another container. 

Folder permissions:

```
 ls -l /data/media/
drwxrwxr-x  3 media media 4096 Mar 31 11:46 unmanic
```

Am I doing something wrong? 

Thank you for your time!

**Comment by groenator** [comment]
It seems that when the container starts it doesn't create the files. The config folder is empty;

```
media@wonkypaw:~$ ls -l /data/media/unmanic/
total 0
```

**Comment by Josh5** [comment]
Thank you for this report and the clear logs.

A fix has been pushed to the master branch 9a59d35

This should be available under the Docker "latest" tag.

**Comment by groenator** [comment]
Thank you, Josh

I will redeploy the image and let you know if have any issues.
